src/engagement_db_coda_sync/lib.py

```diff
@@ -287,15 +287,25 @@ def _update_engagement_db_message_from_coda_message(engagement_db, engagement_db
 
     # WS-correct if there is a valid ws_code
     if ws_code is not None:
+        # Establish the correct dataset to move this message to.
+        # To determine the dataset, the following strategies are tried, in order:
+        #  1. Search the other dataset configurations for a match.
+        #  2. If `set_dataset_from_ws_string_value` has been set, move the message to the dataset `ws_code.string_value`
+        #  3. If the `default_ws_dataset` has been specified, move the message to this default dataset.
+        #  4. Raise a ValueError.
```

Comment:
Should there be an else clause like - "in case we can't find the correct dataset from the config then...`

Reply:
4. was implicitly the else clause - but I'll edit to make it more obvious.

Reply:
I've adjusted this to try to make it clearer